src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -2010,6 +2174,10 @@
   "modifier.tconstruct.dyed.formatted": "染色#%s",
   "modifier.tconstruct.dyed.flavor": "外观改变了！",
   "modifier.tconstruct.dyed.description": "改变盔甲的外观，就好像换了一种制作材料",
+  "modifier.tconstruct.trim": "盔甲纹饰",
+  "modifier.tconstruct.trim.formatted": "%s (%s)",
```

```suggestion
  "modifier.tconstruct.trim.formatted": "%s（%s）",
```
Changed them to full-width variant.